content/en/blog/_posts/get-started-with-kubernetes-using-python.md

```diff
@@ -0,0 +1,217 @@
+﻿---
+layout: blog
+title: Get started with Kubernetes (using Python)
+date: 2019-06-20
+---
+
+**Author**: Jason Haley (Independent Consultant)
+
+So, you know you want to run your application in Kubernetes but don’t know where to start. Or maybe you’re getting started but still don’t know what you don’t know. In this blog you’ll walk through how to containerize an application and get it running in Kubernetes.
+
+This walk-through assumes you are a developer or at least comfortable with the command line (preferably bash shell).
+
+## What we’ll do
+1. Get the code and run the application locally
+2. Create an image and run the application in Docker
+3. Create a deployment and run the application in Kubernetes
+
+## Prerequisites
+* [Docker Desktop with Kubernetes](https://www.docker.com/products/kubernetes) installed (see end of blog for enabling Kubernetes)
+* [Python 3.7](https://www.python.org/) installed
+* [Git](https://git-scm.com/downloads) installed
+
+## Containerizing an application
+In this section you’ll take some source code, verify it runs locally, and then create a Docker image of the application. The sample application used is a very simple Flask web application; if you want to test it locally, you’ll need Python installed. Otherwise, you can skip to the "Create a Dockerfile" section.
+
+### Get the application code
+
+Use git to clone the repository to your local machine:
+```
+git clone https://github.com/JasonHaley/hello-python.git
+```
+
+Change to the app directory:
+```
+cd hello-python/app
+```
+
+There are only two files in this directory. If you look at the main.py file, you’ll see the application prints out a hello message. You can learn more about Flask on the [Flask website](http://flask.pocoo.org/).
+
+```
+from flask import Flask
+app = Flask(__name__)
+
+@app.route("/")
+def hello():
+    return "Hello from Python!"
+
+if __name__ == "__main__":
+    app.run(host='0.0.0.0')
+```
+
+The requirements.txt file contains the list of packages needed by the main.py and will be used by [pip](https://pip.pypa.io/en/stable/) to install the Flask library.
+
+### Run locally
+Manually run the installer and application using the following commands:
+```
+pip install -r requirements.txt
+python main.py
+```
```

Comment:
You may wish to put a note here mentioning that running `pip install` isn't recommended for most scenarios. Instead, it's more common practice to use `virtualenv` (or `pyenv`) and install dependencies within a virtual environment.

Reply:
Ok, once again that shows how much of a beginner I am in Python. Thank you for your review notes, they really add to the quality of the blog!